system/netdata.service.in

```diff
@@ -1,15 +1,56 @@
+# netdata systemd target
+
 [Unit]
-Description=Real time performance monitoring
+Description=netdata - Real-time performance monitoring
+Documentation=man:netdata
+Documentation=file:///usr/share/doc/netdata/html/index.html
```

Comment:
HTML documentation is being installed by some distributions. I'm commenting out the line. Maybe in future it could be introduced again with a standard path.